tests/lint/sconscript

```diff
@@ -47,7 +47,9 @@ lintTarget = externalEnv.Command(
 	"current.diff",
 	[[
 		'type', '$SOURCE', '|',  # provide diff to stdin
-		sys.executable, "-m", "flake8",
+		sys.executable,
+		"-Xutf8",  # UTF-8 mode (PEP 540, Python >= 3.7)

```

Comment:
You'll need to do the same in `appveyor.yml` Look for comment: `# Flake8 Linting`

Reply:
Done: 3153c8d